### DISCUSSION TITLE
ELK Stack connection

### DISCUSSION THREAD

**Discussion by Test83318** [OwlGamingCommunity/V#21] [Support]
What would be the proper way to fetch the logs into elasticsearch?
Filebeat? Logstash? With what configuration? I am not using Docker, if that is important.
Thanks in advance.

**Comment by Yannick-DG** [comment]
Best to use is a Docker instance with ElasticSearch running and shoot the data into the SQL with maybe a POST request? Or write a file on disk and send it towards Elastic afterwards. Filebeat is a good solution too, but as far as I'm aware we didn't have those setup.

**Reply by ChaosParadise** [reply]
We actually did use filebeat. This server writes logs to a particular directory and we would collect them with filebeat and send to ES

**Reply by Yannick-DG** [reply]
Possibly you've got a config for the lad unless it's private?
